platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskAssignInChatTestIT.java

```diff
@@ -51,13 +55,17 @@ public void setupBeforeMethod() {
   @Tag("TA-609")
```

Comment:
The bug TA-609 is fixed. Do you usually keep the tag in this case ? Doesn't it ignore automatically the test ?

Reply:
Yes we keep the tag of the Jira Issue Id to get a quick idea about the functional aspect of test case.

Reply:
ok :+1: